sites/all/modules/custom/training5_file_lib/training5_file_lib.pages.inc

```diff
@@ -0,0 +1,194 @@
+<?php
+
+/**
+ * @file
+ * Implements page callbacks for training5_file_lib.module.
+ */
+
+/**
+ * Url with current exchange rates.
+ */
+define('TRAINING5_FILE_LIB_NBRB_RATES_XML', 'http://www.nbrb.by/Services/XmlExRates.aspx');
+
+/**
+ * Page callback for files.
+ */
+function training5_file_lib_files() {
+  global $user;
+  ctools_include('modal');
+  ctools_include('ajax');
+  ctools_modal_add_js();
+  drupal_add_css(drupal_get_path('module', 'training5_file_lib') . '/css/training5_file_lib.css');
```

Comment:
Code duplication for drupal_get_path('module', 'training5_file_lib'). 
The function calls twice.